cloudinit/net/__init__.py

```diff
@@ -721,7 +721,7 @@ def _rename_interfaces(
 
     LOG.debug("Detected interfaces %s", cur_info)
 
-    def update_byname(bymac):
+    def update_byname(cur_info):
```

Comment:
Yes.

Reply:
Huh, this is funny. This was "broken" in https://github.com/canonical/cloud-init/commit/ebc9ecbc8a76bdf511a456fb72339a7eb4c20568, which forgot to change the argument name from `bymac` to `cur_info` except that it didn't change the behavior at all, since the variable `cur_info` is already in scope when the function is defined so it just changed to being a closure.

I think this change is correct, but I don't like that it shadows a variable of the same name in the outer scope. Could you please use a different argument name than `cur_info` to make it clear that this isn't a closure?